typescript/schedules.md

```diff
@@ -0,0 +1,1006 @@
+# Schedules
+
+- [Docs](https://docs.temporal.io/workflows/#schedules)
+- gRPC API:
+  - [methods](https://github.com/temporalio/api/blob/799926c86eb13d8a9717d3561ab9b0df43796c06/temporal/api/workflowservice/v1/service.proto#L328-L370)
+  - [`request_response.proto`](https://github.com/temporalio/api/blob/799926c86eb13d8a9717d3561ab9b0df43796c06/temporal/api/workflowservice/v1/request_response.proto#L821-L957)
+  - [`schedule/v1/message.proto`](https://github.com/temporalio/api/blob/master/temporal/api/schedule/v1/message.proto)
+  - [`ScheduleOverlapPolicy` enum](https://github.com/temporalio/api/blob/master/temporal/api/enums/v1/schedule.proto)
+
+## TS API
+
+```ts
+import { ScheduleClient, ScheduleOverlapPolicy } from '@temporalio/client'
+import { addWeeks, subDays } from 'date-fns'
+import { myWorkflow } from './workflows'
+
+const client = new ScheduleClient();
+
+const schedule = await client.create({
+  id: 'schedule-biz-id',
+  spec: {
+    // Schedule fires at combination of all `intervals` and `calendars`, minus the `skip` calendar:
+    intervals: [
+      // every hour at minute 5
+      {
+        every: '1h',
+        offset: '5m',
+      },
+      // every 20 days since epoch at day 2
+      {
+        every: '20d',
+        offset: '2d',
+      },
+    ],
+    calendars: [{
+      // at noon on the 1st, 3rd, 5th, 7th, 9th, and 22nd days of the month
+      hour: 12,
+      dayOf: [{
+        start: 1,
+        end: 10,
+        step: 2,
+      }, 22],
+    }],
+    skip: [
+      {
+        // skip 11:05 pm daily
+        hour: 23,
+        minute: 5,
+      },
+    ],
```

Comment:
Alternative:

```suggestion
    skip: [
      {
        // skip 11:05 pm daily
        hour: [23],
        minute: [5],
      },
    ],
```

- Pro: Simplifies the type (from `Hour | Hour[] | '*'` to `Hour[] | '*'`.
- Con: Often have to type unnecessary `[]` characters.